example/legacy/unstake_token.dart

```diff
@@ -5,11 +5,11 @@ import 'package:fuse_wallet_sdk/fuse_wallet_sdk.dart';
 import 'create_wallet.dart';
 
 void main() async {
-  final String privateKey = await Mnemonic.generatePrivateKey();
-  final EthPrivateKey credentials = EthPrivateKey.fromHex(privateKey);
+  final privateKey = await Mnemonic.generatePrivateKey();
```

Comment:
If we generate a new user the example should fail, no?
In this case it's better to use env vars so user will understand that he should provide a PK for wallet with staked funds